data-es6.js

```diff
@@ -2914,6 +2914,30 @@ exports.tests = [
   },
 },
 {
+  name: 'prototype of bound functions',
+  category: 'functions',
+  significance: 'small',
+  link: 'https://people.mozilla.org/~jorendorff/es6-draft.html#sec-boundfunctioncreate',
+  exec: function () {/*
+    function getProtoBound(proto) {
+      var f = Object.setPrototypeOf(function() { }, proto)
+      var boundF = Function.prototype.bind.call(f, null)
+      return Object.getPrototypeOf(boundF)
+    }
+
+    return [ Function.prototype, function(){}, [], null ].every(function(proto) {
```

Comment:
I don't see the necessity of the distinction. If you want to know "does the implementation implement the correct semantics for bound functions" then you need to assume generators exist, because the correct semantics for bound functions *require the existence of generators*. If generator functions do not exist, then any semantics involving them are obviously not implemented. (If you instead want to know "does the implementation implement the correct semantics for bound *normal* functions" then the specific subtest provides complete information.)

I also don't see the necessity of the highly misleading "N/A" answer. If the bound generator functions test says "No", then you can find out whether basic generator support exists by checking the basic generator function test result. Either way, it simply tells the truth: no, the platform does not support feature X used with feature Y. Remember that in order to be ES6 compliant, *the entire feature set*, barring Annex B where noted, must be supported - not implementing every feature that intersects with a feature is leaving that feature incomplete.